mmdet/models/losses/focal_loss.py

```diff
@@ -19,7 +20,8 @@ def py_sigmoid_focal_loss(pred,
     Args:
         pred (torch.Tensor): The prediction with shape (N, C), C is the
             number of classes
-        target (torch.Tensor): The learning label of the prediction.
+        target (torch.Tensor): The learning label of the prediction,
```

Comment:
It's not a good idea to change the argument type of `py_sigmoid_focal_loss`. This function could be used in other downstream codebases, though it's not called in mmdetection.